### PR TITLE
Theming fixes stable9

### PR DESCRIPTION
**Pull request by nickvergessen**
Backport of #405 

cc @MorrisJobke @LukasReschke @schiessle

### CODE REVIEW

File: apps/theming/lib/template.php [hunk 1]
```diff
@@ -93,6 +97,15 @@ public function getSlogan() {
 		return $this->config->getAppValue('theming', 'slogan', $this->slogan);
 	}
 
+	public function getShortFooter() {
+		$slogan = $this->getSlogan();
+		$footer = '<a href="'. $this->getBaseUrl() . '" target="_blank"' .
+			' rel="noreferrer">' .$this->getEntity() . '</a>'.
+			($slogan !== '' ? ' – ' . $slogan : '');
```
Comment (LukasReschke) [inline apps/theming/lib/template.php]: Unit test? 😉

Comment (nickvergessen) [inline apps/theming/lib/template.php]: done

**Comment by MorrisJobke** [comment]
Tested and works 👍

**Comment by schiessle** [comment]
looks good :+1:

**Pull request merged**